viewers/nerf_viewer/bridge_server/.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
What if we merge this file with the [.gitignore](https://github.com/openxrlab/xrprimer/blob/main/.gitignore) file in the root directory?

Reply:
The original .gitignore file is for debug purpose only. It has been removed now.